app/controllers/claims_controller.rb

```diff
@@ -23,10 +23,13 @@ def show
     if params[:slug] == "postcode-search" && postcode
       redirect_to claim_path(current_policy_routing_name, "select-home-address", {"claim[postcode]": params[:claim][:postcode], "claim[address_line_1]": params[:claim][:address_line_1]}) and return unless invalid_postcode?
     elsif params[:slug] == "select-home-address" && postcode
+      session[:claim_postcode] = params[:claim][:postcode]
+      session[:claim_address_line_1] = params[:claim][:address_line_1]
```

Comment:
Will this param ever be present? We might be introducing a new bug otherwise (where the user has manually entered their address). Suggest removing this line.

Reply:
Yeah it's used, this fixes a bug where pressing back the form picks a previous stale session value if the user goes forward and back more than once.